rust/fory-derive/src/object/util.rs

```diff
@@ -768,7 +768,12 @@ pub(crate) fn get_sorted_field_names(fields: &[&Field]) -> Vec<String> {
 }
 
 pub(super) fn get_sort_fields_ts(fields: &[&Field]) -> TokenStream {
-    let sorted_names = get_sorted_field_names(fields);
+    let filterd_fields: Vec<&Field> = fields
```

Comment:
Could we extract a method to get filtered fields Vec And reuse it in read and write.rs